src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommandParser.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class InstallToolCommandParser
+    {
+        public static Command InstallTool()
+        {
+            return Create.Command("tool",
+                "Install tool",
+                Accept.ExactlyOneArgument(o => "packageId")
```

Comment:
Should we have suggestions here?

Reply:
Yes, I missed this string to let Kathleen review. She is oof this week. Can I have an issue to track this and let her review it later

Reply:
https://github.com/dotnet/cli/issues/8162